shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -145,7 +147,11 @@ private Status check() {
   public DistributedLoadCommand(FileSystemContext fsContext) {
     super(fsContext);
     mSubmittedJobAttempts = Lists.newArrayList();
-    mClient = null;
+
+    InstancedConfiguration conf = new InstancedConfiguration(ConfigurationUtils.defaults());
+    final ClientContext clientContext = ClientContext.create(conf);
+    mClient = JobMasterClient.Factory.create(
```

Comment:
Can you override the Command#close() and close this client there?

Reply:
oops. You are right. I forgot. Done now.